cmake/Reprojucer.cmake

```diff
@@ -345,19 +351,26 @@ function(jucer_project_module module_name PATH_TAG modules_folder)
   foreach(src_file ${module_src_files})
     unset(to_compile)
 
-    # See LibraryModule::CompileUnit::isNeededForExporter()
```

Comment:
Was the comment intentionally removed?

Reply:
Yes. The code moved a bit between JUCE 4 and JUCE 5, making the comment not completely accurate.